yapapi/executor/ctx.py

```diff
@@ -239,8 +239,8 @@ def run(
         :param env: optional dictionary with environmental variables
         :return: None
         """
-        stdout = CaptureContext.build()
-        stderr = CaptureContext.build()
+        stdout = CaptureContext.build(mode="stream")
```

Comment:
This will cause exe-unit to emit events with the messages that the running process outputs to its stdout and stderr